static/src/javascripts/projects/common/modules/identity/upsell/upsell.js

```diff
@@ -28,38 +27,9 @@ const bindAccountCreation = (el) => {
     });
 };
 
-
-const getPrefill = (el) => ({
-    csrfToken: el.dataset.csrfToken,
-    accountToken: el.dataset.accountToken,
-    email: el.dataset.email,
-    hasPassword: el.dataset.hasPassword === 'true',
-    hasSocialLinks: el.dataset.hasSocialLinks === 'true',
-});
-
-const bindBlockList = (el) => {
-    fastdom
-        .measure(() => getPrefill(el))
-        .then(prefill =>
-            fastdom.mutate(() => {
-                render(
-                    <StatefulConfirmEmailPage
-                        csrfToken={prefill.csrfToken}
-                        accountToken={prefill.accountToken}
-                        email={prefill.email}
-                        hasPassword={prefill.hasPassword}
-                        hasSocialLinks={prefill.hasSocialLinks}
-                    />,
-                    el
-                );
-            })
-        );
-};
-
 const enhanceUpsell = () => {
     loadEnhancers([
         ['.js-identity-upsell-account-creation', bindAccountCreation],
-        ['.js-identity-block-list', bindBlockList],
```

Comment:
deleting this enhancer allows us to remove all the JavaScript code
`.js-identity-block-list` no longer exists in the project. It was probably removed in another PR and replaced with flows in the manage my account or Gateway projects.